javacutil/src/main/java/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -893,4 +894,24 @@ public static TypeMirror substitute(
                 com.sun.tools.javac.util.List.from(newP),
                 com.sun.tools.javac.util.List.from(newT));
     }
+
+    // This method is necessary because Type.stripMetadata does no work for primitive types.
+    /**
+     * Returns a copy of a type without annotations. May return tm itself if it has no annotations.
+     *
+     * @param tm a TypeMirror
+     * @return a copy of the given type without annotations
+     */
+    public static TypeMirror unannotatedType(TypeMirror tm) {
```

Comment:
There's another version of this method `org.checkerframework.javacutil.TypeAnnotationUtils#unannotatedType`.  It makes more sense to have this method in TypesUtils.  Or at least that's where I always look for it first.